text/0000-clarify-api-namespace.md

```diff
@@ -0,0 +1,41 @@
+- Start Date: 2015-08-10
+- RFC PR: (leave this empty)
```

Comment:
This and RFC number update is done only after the PR is merged by Stephen. So, I will leave it.